Dockerfile

```diff
@@ -1,63 +1,51 @@
-FROM node:16 as sourcebuild
-ARG ui_tag=main
-ARG api_tag=main
-
+# Frontend build
+FROM node:16 as FRONTEND
 WORKDIR /app
-RUN git clone --depth 1 --branch ${ui_tag} https://github.com/KaotoIO/kaoto-ui.git 
-RUN git clone --depth 1 --branch ${api_tag} https://github.com/KaotoIO/kaoto-backend.git 
-
+RUN git clone --depth 1 https://github.com/KaotoIO/kaoto-ui.git 
 WORKDIR /app/kaoto-ui
-RUN yarn install --mode=skip-build
-
-RUN KAOTO_API="" yarn run build
```

Comment:
this cannot be removed. It overrides the default defined in webpack config.

Reply:
The `skip-build` or the `KAOTO_API`?

KAOTO_API is restored on line 7. Maybe I should add skip-build too.